frontend/src/components/dashboard/create/Create.module.scss

```diff
@@ -130,8 +130,12 @@
   font-style: italic;
   color: $secondary;
   text-decoration: underline;
-  &.infoLink {
-    color: $primary;
+  color: $primary;
```

Comment:
Duplicated `color` as above. Might want to consider `@extend %body-1` as specified in Figma instead of setting values for `font-size` and `font-weight`

![image](https://user-images.githubusercontent.com/3666479/99626898-2b877980-2a6e-11eb-8ef3-124a20f6363d.png)

Reply:
Removed `font-size` and `font-weight` as it was inherited. Removed the margin as it was not necessary.